### PR TITLE
hw14 commit

### PR DESCRIPTION
**Pull request by iibrahimsv**
hw14 commit

### CODE REVIEW

**Comment by iibrahimsv** [comment]
Homework 14 is ready to be checked!

File: src/test/java/hw14.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+public class hw14 {
+
+    @Test
+    public static void RegistrationPage () throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
+        try {
+            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+            String url = "https://bbb.testpro.io/";
+            driver.get(url);
```
Comment (Berichicko) [inline src/test/java/hw14.java:18]: Please, adding Java variable

For, example
`String registrationURL = "https://bbb.testpro.io/registration.php";`

File: src/test/java/hw14.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            WebElement registrationField = driver.findElement(By.cssSelector("[id = 'hel']"));
+            registrationField.click();
+            WebElement registrationButton = driver.findElement(By.cssSelector("[id = 'button']"));
+            Assert.assertTrue(registrationButton.isDisplayed());
```
Comment (Berichicko) [inline src/test/java/hw14.java:22]: Please, assert if the current url is equal to your stored (String)

For, example 

`Assert.assertEquals(driver.getCurrentUrl(), registrationURL);`

File: src/test/java/hw14.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class hw14 {
+
+    @Test
+    public static void RegistrationPage () throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
+        try {
```
Comment (Berichicko) [inline src/test/java/hw14.java:15]: You can: to maximize browser in Selenium, you need to call the maximize() Selenium command to maximize window interface of the driver class.

`driver.manage().window().maximize();`

**Comment by Berichicko** [comment]
Hello

Homework  #14 is cheeked! 

I left some recommendations for the code.🔥

**Well, great job**! 🔥


**Your grade: 95** ✊


💖Best wishes, the team HW (Alex)!💖

**Pull request closed without merging**